monai/apps/deepgrow/interaction.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Callable, Dict, Sequence, Union
+
+import torch
+
+from monai.engines import SupervisedEvaluator, SupervisedTrainer
+from monai.engines.utils import CommonKeys
+from monai.engines.workflow import Events
+from monai.transforms import Compose
+
+
+class Interaction:
+    """
+    Ignite handler used to introduce interactions (simulation of clicks) for Deepgrow Training/Evaluation.
```

Comment:
please add paper references to the docstring

Reply:
fixed